templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
Ahaha, sorry, you're right I'm not clear!
When you click the button, the popover opens, you can select an option and it closes right away, which is perfect!
BUT, when after that you try to click te button again, it doesnt work. You need to click just under the icon. It doesn't work after that.